config/api.yml

```diff
@@ -2869,6 +2869,38 @@
         :identifier: miq_request_control
       - :name: edit
         :identifier: miq_request_edit
+  :pxe_image_types:
+    :description: PXE Image Types
+    :identifier: pxe_image_type_accord
+    :options:
+    - :collection
+    :verbs: *gpd
```

Comment:
Should this be `gpppd`?  @abellotti ?

Reply:
It can be, essentially adding the PATCH and PUT, if we do, then the specs should be extended to include those, that's all.

Reply:
What is the criteria for adding `PATCH` and `PUT`? Just curious.

Reply:
I generally think we should do post/patch/put as a group.  I think the better question is "What is the criteria for NOT adding `PATCH` and `PUT`"?